src/ExternalSearch.Providers.Gleif/GleifExternalSearchJobData.cs

```diff
@@ -1,23 +1,24 @@
 ﻿using System.Collections.Generic;
 using CluedIn.Core.Crawling;
+using static CluedIn.ExternalSearch.Providers.Gleif.Constants;
 
 namespace CluedIn.ExternalSearch.Providers.Gleif
 {
     public class GleifExternalSearchJobData : CrawlJobData
     {
         public GleifExternalSearchJobData(IDictionary<string, object> configuration)
         {
-            AcceptedEntityType = GetValue(configuration, nameof(AcceptedEntityType), default(string));
-            LeiVocabularyKey = GetValue(configuration, nameof(LeiVocabularyKey), default(string));
-            SkipEntityCodeCreation = GetValue(configuration, nameof(SkipEntityCodeCreation), default(bool));
+            AcceptedEntityType = GetValue(configuration, KeyName.AcceptedEntityType, default(string));
+            LeiVocabularyKey = GetValue(configuration, KeyName.LeiVocabularyKey, default(string));
+            SkipEntityCodeCreation = GetValue(configuration, KeyName.SkipEntityCodeCreation, default(bool));
         }
 
         public IDictionary<string, object> ToDictionary()
         {
             return new Dictionary<string, object> {
-                { nameof(AcceptedEntityType), AcceptedEntityType },
-                { nameof(LeiVocabularyKey), LeiVocabularyKey },
-                { nameof(SkipEntityCodeCreation), SkipEntityCodeCreation },
+                { KeyName.AcceptedEntityType, AcceptedEntityType },
```

Comment:
the old nameof(...) would have returned the value AcceptedEntityType not the camel case that this new version returns. Have you tested if this will cause an issue for a customer upgrading?

Reply:
tested, it's case insensitive